addons/binding/org.openhab.binding.zway/README.md

```diff
@@ -153,6 +155,10 @@ Because textual configuration isn't useful, follow the instructions in the [Gett
 
 ## Developer stuff
 
+### Known issues
+
+- The Z-Way Binding only works, when simple mode of item linking is enabled during thing creation.
```

Comment:
Why this? This is a major flaw, if this is required... Is it because you rely on a specific name of the items? Couldn't this be refactored in the code?